.github/workflows/deb-release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release deb package
```

Comment:
could you please either refactor that flow into this file https://github.com/cargo-generate/cargo-generate/blob/main/.github/workflows/release-binary-assets.yml
as a dedicated job, 

or change the trigger that it's triggered from that named release-binary-assets workflow.

Reply:
sure i will try to add it to the `release-binary-assets.yml` workflow